leetcode/581.shortest-unsorted-continuous-subarray.py

```diff
@@ -0,0 +1,28 @@
+class Solution(object):
+    def findUnsortedSubarray(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        n = len(nums)
+        l, i = 0, 0
+        for i in range(1, n):
+            if nums[i]<nums[i-1]:
+                l = i
+                break
+
+        for i in range(i, n):
+            while l>=0 and nums[i]<nums[l]:
```

Comment:
```
Input: [2,6,4,8,10,9,15]
Output: 7
Expected: 5
```
should be `nums[i]<nums[l-1]`, continue in while loop, compare nums[i] with nums[l-1] first and then decide whether it's a new boundary, not l, as l is already determined.